sources/src/main/java/com/ullink/slack/simpleslackapi/impl/SlackWebSocketSessionImpl.java

```diff
@@ -833,24 +782,20 @@ public void setPresence(SlackPersona.SlackPresence presence) {
 
     }
 
-    private synchronized long getNextMessageId()
```

Comment:
Don't need to do this. Removed synchronized from the method and changed messageId variable as AtomicLong.